src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsCompactChat.kt

```diff
@@ -57,7 +60,12 @@ object HoppityEggsCompactChat {
 
         return if (duplicate) {
             val format = lastDuplicateAmount?.let { NumberUtil.format(it) } ?: "?"
-            "$mealName Egg! §7Duplicate $lastName §7(§6+$format Chocolate§7)"
+            val timeHumanized = lastDuplicateAmount?.let {
```

Comment:
please use the wording "formatted" instead of "humanized" for consistency

Reply:
Fixed in [689d42b](https://github.com/hannibal002/SkyHanni/pull/1887/commits/689d42bbd824c85181237c50d7003f36fca5a8ca)